src/Sentry/Internal/DistributionLocator.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Sentry.Internal
+{
+    internal static class DistributionLocator
```

Comment:
wouldnt this be simpler with a `GetEnvironmentVariable` in the static constructor and allow the resulting prop to be settable for testing purposes?

Reply:
It sure would.  I was copying the existing pattern from the other "locator" classes, but that would be much simpler.  I'll do that next.

Reply:
We can do much better with testing on environment variables anyway, and those locators can be collapsed to a single class.

Reply:
On second thought, I'm going to do that work in a separate PR first and then update this to use it later.  We'll hold this one for now.